src/services/agent-service.ts

```diff
@@ -109,6 +109,8 @@ export class AgentService {
       minHeight: request.body.minHeight || this.snapshotConfig['min-height'],
     }
 
+    snapshotLogger.debug(`-> snapshot options: ${JSON.stringify(snapshotOptions)}`)
```

Comment:
Do we want to use JSON or inspect? Just a preference, JSON wraps keys in strings and we use inspect [here](https://github.com/percy/percy-agent/blob/master/src/utils/configuration.ts#L93). Inspect will keep things single line until a certain length, while JSON is all or nothing with the third parameter.